dali/pipeline/operators/color/hsv.h

```diff
@@ -0,0 +1,281 @@
+// Copyright (c) 2019, NVIDIA CORPORATION. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef DALI_PIPELINE_OPERATORS_COLOR_HSV_H_
+#define DALI_PIPELINE_OPERATORS_COLOR_HSV_H_
+
+#include <vector>
+#include <memory>
+#include <string>
+#include "dali/core/static_switch.h"
+#include "dali/core/geom/mat.h"
+#include "dali/pipeline/data/views.h"
+#include "dali/pipeline/operators/common.h"
+#include "dali/pipeline/operators/operator.h"
+#include "dali/kernels/kernel_manager.h"
+#include "dali/kernels/tensor_shape_print.h"
+#include "dali/kernels/imgproc/pointwise/linear_transformation_cpu.h"
+
+
+namespace dali {
+namespace hsv {
+
+
+const std::string kHue = "hue_delta";                // NOLINT
```

Comment:
It will break the moment this header is included in multiple translation units - the linker will fail with multiply defined symbol.
Also, why hide the name behind this constant? Not to mention, that one-character names "H", "S", "V" would be perfectly acceptable for the user (and certainly the _delta suffix should be dropped).

Reply:
> Also, why hide the name behind this constant?

Precisely for the reason of comments like these ;) You want me to change this, I change it in one place, not 3 or 4 places, where it's used. Solely for this reason it can be constant.

As for arg names, AFAIK python code style stipulates arg names to be snake case, so I'd rather go for `hue` etc.